app/models/manageiq/providers/vmware/infra_manager/host_esx.rb

```diff
@@ -2,6 +2,30 @@ class ManageIQ::Providers::Vmware::InfraManager::HostEsx < ManageIQ::Providers::
   supports :refresh_advanced_settings
   supports :refresh_firewall_rules
   supports :refresh_logs
+  supports :reboot do
+    validate_active_with_power_state(:reboot, "on")
+  end
+  supports :shutdown do
+    validate_active_with_power_state(:shutdown, "on")
+  end
+  supports :standby do
+    validate_active_with_power_state(:standby, "on")
+  end
+  supports :enter_maint_mode do
+    validate_active_with_power_state(:enter_maint_mode, "on")
+  end
+  supports :exit_maint_mode do
+    validate_active_with_power_state(:exit_maint_mode, "maintenance")
+  end
+  supports :enable_vmotion do
+    validate_active_with_power_state(:enable_vmotion, "on")
+  end
+  supports :disable_vmotion do
+    validate_active_with_power_state(:disable_vmotion, "on")
+  end
+  supports :vmotion do
+    validate_active_with_power_state(:vmotion, "on")
+  end
```

Comment:
This one is a little weird, it really means "am I able to ask if vmotion is enabled" not actually checking if it is enabled might want to do this differently